apps/dashboard/src/app/(app)/team/[team_slug]/[project_slug]/insight/webhooks/components/_components/FilterDetailsStep.tsx

```diff
@@ -0,0 +1,543 @@
+"use client";
+
+import { MultiNetworkSelector } from "@/components/blocks/NetworkSelectors";
+import { Spinner } from "@/components/ui/Spinner/Spinner";
+import { Badge } from "@/components/ui/badge";
+import { Button } from "@/components/ui/button";
+import {
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@/components/ui/form";
+import { Input } from "@/components/ui/input";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+import { Textarea } from "@/components/ui/textarea";
+import { useThirdwebClient } from "@/constants/thirdweb.client";
+import { cn } from "@/lib/utils";
+import { useQueryClient } from "@tanstack/react-query";
+import type { UseFormReturn } from "react-hook-form";
+
+import { truncateMiddle } from "../_utils/abi-utils";
+import type {
+  AbiData,
+  EventSignature,
+  FunctionSignature,
+  WebhookFormValues,
+} from "../_utils/webhook-types";
+
+interface FilterDetailsStepProps {
+  form: UseFormReturn<WebhookFormValues>;
+  eventSignatures: EventSignature[];
+  functionSignatures: FunctionSignature[];
+  fetchedAbis: Record<string, AbiData>;
+  abiErrors: Record<string, string>;
+  fetchedTxAbis: Record<string, AbiData>;
+  txAbiErrors: Record<string, string>;
+  isFetchingEventAbi: boolean;
+  isFetchingTxAbi: boolean;
+}
+
+export default function FilterDetailsStep({
```

Comment:
no `export default` ever unless required by next.js for pages/layouts similar

@MananTank if we can add a rule for this into lint then please lets do so